src/SFA.DAS.EmployerIncentives/Configuration/AzureManagedIdentityApiConfiguration.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.EmployerIncentives.Configuration
+{
+    public class AzureManagedIdentityApiConfiguration
```

Comment:
Class not used - can it be deleted please

Reply:
deleted